docs/CHANGES.TXT

```diff
@@ -1,5 +1,6 @@
 0.95 (to be released)
 -----------------
+- Fix: respect `-stdout` even if multiple CC tracks are present in a Matroksa input file (#1453)
```

Comment:
Typo in Matroska

Reply:
Fixed in 7f91df5403ff4fcf58461ae0d9ef559f10ea6c6e. Thanks!